code/game/gamemodes/clock_cult/clock_objects.dm

```diff
@@ -1362,12 +1362,6 @@
 			if(src)
 				qdel(src)
 
-/obj/effect/clockwork/spatial_gateway/Destroy()
-	visible_message("<span class='warning'>[src] flickers and closes!</span>")
-	animate(src, transform = matrix() - matrix(), time = 10)
-	spawn(10)
-		..()
```

Comment:
I'm probably actually going to remove the spawn animation, too. I decided that it might cause some confusion as to how the portal was functioning (that may sound ridiculous, but learning things for the first time on Space Station 13 is much different than learning them anywhere else).